### DISCUSSION TITLE
Troubles with github students

### DISCUSSION THREAD

**Discussion by 1ilir0lika** [community/community#42927] [GitHub Education]
Hi I'm an italian student and last year I tried to use github students,they rejected me because the text wasn't readable,but I have to take a picture of me,the print of the original document and a print of its translation with my phone and it's quite hard,can i send the single documents and then a photo of me holding the original?
P.s. I'm in second year of high school

**Comment by myConsciousness** [comment]
Hi @1ilir0lika ,

I guess, this seems like a question that should be pitched to GitHub's support desk for this plan.

Or, you can check this:

- https://support.github.com/?tags=dotcom-direct

**Comment by jfullstackdev** [comment]
there are similar issues like that here on GitHub Community

other posts were responded by the GitHub Staff, though there is still no assurance,

you may want to try Support too 

https://support.github.com/contact/account
